cli/module_registry.go

```diff
@@ -252,10 +252,17 @@ func UploadModuleAction(c *cli.Context) error {
 			return errors.Errorf("module name %q was supplied on the command line but the meta.json has a module ID of %q", nameArg,
 				moduleID.name)
 		}
-	}
-	moduleID, err = validateModuleID(client, moduleID.String(), publicNamespaceArg, orgIDArg)
-	if err != nil {
-		return err
+
+		moduleID, err = validateModuleID(client, moduleID.String(), publicNamespaceArg, orgIDArg)
+		if err != nil {
+			return err
+		}
+
+		_, err = client.updateModule(moduleID, manifest)
+		if err != nil {
+			printf(c.App.Writer, "Module update failed. Please correct the following issues in your meta.json:")
+			return err
```

Comment:
```suggestion
			return errors.Wrap(err, "Module update failed. Please correct the following issues in your meta.json")
```
+ a nolint
The print error thing seems fragile